Jellyfin.Drawing.Skia/SkiaEncoder.cs

```diff
@@ -233,7 +229,7 @@ private string NormalizePath(string path)
                 return path;
             }
 
-            var tempPath = Path.Combine(_appPaths.TempDirectory, Guid.NewGuid() + Path.GetExtension(path) ?? string.Empty);
+            var tempPath = Path.Combine(_appPaths.TempDirectory, Guid.NewGuid() + Path.GetExtension(path));
```

Comment:
care to explain?

Reply:
GetExtension only returns null when path is null